Modules/Test/classes/class.ilObjTest.php

```diff
@@ -4926,6 +4930,65 @@ function getUnfilteredEvaluationData()
 					}
 				}
 			}
+			else if($this->isDynamicTest())
+			{
+				require_once 'Modules/Test/classes/class.ilTestSequenceFactory.php';
```

Comment:
You can delete the require_once's due to the classmap

Reply:
That's true for ILIAS >= 5.3.x, but this is not a good idea if Björn wants to cherry-pick this as a fix to branches of maintained versions.